README.md

```diff
@@ -1,14 +1,173 @@
 # formfiler
 [![Build Status](https://travis-ci.org/diy-asylum/formfiller.svg?branch=master)](https://travis-ci.org/diy-asylum/formfiller)
+[![Coverage Status](https://coveralls.io/repos/github/diy-asylum/formfiller/badge.svg?branch=master)](https://coveralls.io/github/diy-asylum/formfiller?branch=master)
+
```

Comment:
Example coverage report
https://coveralls.io/builds/20752830

Reply:
Example build:
https://travis-ci.org/diy-asylum/formfiller/builds/470820158
and its corresponding build scan
https://gradle.com/s/qlsurhdygnsxu